tests/jepsen.clickhouse-keeper/src/jepsen/clickhouse_keeper/utils.clj

```diff
@@ -45,7 +45,7 @@
 
 (defn zk-connect
   [host port timeout]
-  (exec-with-retries 30 (fn [] (zk/connect (str host ":" port) :timeout-msec timeout))))
+  (zk/connect (str host ":" port) :timeout-msec timeout))
```

Comment:
Jepsen tries to create clients again on the workers so retries are only necessary for the cases when we need to make sure to connect.
For other cases, if we have a crash on a server, jepsen cannot finish until all workers are done, and with retries this will be true only after `30 * timeout-msec` which can make tests run for 2 hours or more.

Reply:
not sure what you mean by this comment 😄